api/room/schema_query.py

```diff
@@ -211,3 +232,35 @@ def resolve_bookings_analytics_count(self, info, start_date, end_date):  # noqa:
         analytics = RoomAnalyticsRatios.get_bookings_analytics_count(
             self, query, start_date, end_date)
         return analytics
+
+    def resolve_analytics_for_daily_room_events(self, info,
+                                                start_date, end_date
+                                                ):
+        start_date, end_date = CommonAnalytics().convert_dates(
```

Comment:
Why is it necessary to include the brackets to the CommonAnalytics class?

Reply:
This is to ensure I have created an object of the CommonAnalytics class. This way I do not use the method as a static function.